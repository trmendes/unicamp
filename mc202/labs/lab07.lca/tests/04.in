16 21
lca 1 1
lca 2 2
lca 3 3
lca 4 4
lca 15 15
lca 16 16
link 16 15
cut 16
link 2 1
link 3 1
link 4 1
link 5 1
lca 1 2
lca 2 1
lca 3 1
link 16 15
link 14 15
lca 14 16
lca 15 14
cut 14
lca 14 14
