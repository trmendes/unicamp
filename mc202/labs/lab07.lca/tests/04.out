1
2
3
4
15
16
1
1
1
15
15
14
